docs/UserGuide.md

```diff
@@ -445,35 +441,35 @@ Examples:
 * `list` displays all students and lessons in TutorAid by only showing their names and list indexes.
 * `list -a` displays all students and lessons in TutorAid while showing all of their fields' data.
 
-### Adding students to lessons: `add -sl`
+### Adding student to lesson: `add -sl`
```

Comment:
Since this is for 2101... Adding a student to a lesson? 😆

Reply:
Fixed, thanks! 😂 Updated the command description for `del -sl` too